scripts/zkai_proof_pressure_scaling_claim_pack_gate.py

```
@@ -1132,37 +1123,51 @@ def validate_scale_signal(payload: dict[str, Any]) -> None:
-        raise ProofPressureScalingClaimPackError("d32 fused raw byte overclaim")
-    if d32.get("fused_binary_raw_status") != FULLER_ROUTE_FUSED_BINARY_RAW_STATUS:
-        raise ProofPressureScalingClaimPackError("d32 fused raw status drift")
-    if d32.get("source_plus_sidecar_raw_proof_bytes") != 116_682:
-        raise ProofPressureScalingClaimPackError("d32 comparator metric drift")
-    if d32.get("fused_to_source_plus_sidecar_ratio") is not None:
-        raise ProofPressureScalingClaimPackError("d32 ratio overclaim")
-    if d32.get("fused_to_source_plus_sidecar_ratio_status") != FULLER_ROUTE_MIXED_RATIO_STATUS:
-        raise ProofPressureScalingClaimPackError("d32 ratio status drift")
+
+
+def validate_route_matrix_signal(payload: dict[str, Any]) -> None:
```

Ruff (0.15.13)

[warning] Too many branches (20 > 12)

(PLR0912)